app/views/madmin/users/_delete_button.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= button_to path,
+    method: :delete,
+    class: "btn btn-outline-danger btn-sm #{'d-flex align-items-center' if show_icon}",
+    data: {
+      turbo: true,
+      turbo_confirm: _("⚠️ WARNING: Deleting %{email} will also permanently delete ALL %{count} push(es) created by this user. This action cannot be undone. Are you sure you want to proceed?") %
+      { email: record.email, count: push_count }
+    } do %>
+  <% if show_icon %>
```

Comment:
Less and more simple code is better.  Let's just simplify and just show the icons on all the delete/view buttons in both interfaces.